internal/cmd/destinations.go

```diff
@@ -83,7 +84,7 @@ func newDestinationsRemoveCmd() *cobra.Command {
 			}
 
 			if len(destinations) == 0 {
-				return fmt.Errorf("no destinations named %s", args[0])
+				return fmt.Errorf("No destinations named %s.", args[0])
```

Comment:
errors should start with a lowercase character

Reply:
Ok. If we displayed all messages as `fmt.Printf("Error: %s\n", err)`, the lowercase rule would work.

Reply:
done